lib/lab_manager/provider/vsphere.rb

```diff
@@ -94,7 +103,7 @@ def create_vm(opts = {})
 
       VSphere.with_connection do |vs|
         dest_folder = opts[:dest_folder]
-        vm_name = opts[:name] || 'lm_' + SecureRandom.hex(8)
+        vm_name = opts[:name] || "#{compute.image}-#{SecureRandom.hex(8)}"
```

Comment:
I would then suggest to use something like YYYYMMDD-hhmmssxxx-guid to ensure uniqueness